docs/api/localij.md

```diff
@@ -0,0 +1,44 @@
+# Local IJ coordinate functions
+
+These functions operate on a local IJ coordinate space. The
+coordinates are anchored by an **origin index**. The I and J
+axes are 120&deg; apart. The origin index might not be at
+(0, 0) in the local coordinate system.
+
+Local IJ coordinates are only valid within a certain radius of the origin.
+The coordinate space used by these functions may have deleted regions or
+warping due to pentagonal distortion.
+
+## experimentalH3ToLocalIj
+
+```
+int experimentalH3ToLocalIj(H3Index origin, H3Index h3, CoordIJ *out);
+```
+
+Produces local IJ coordinates for an H3 index anchored by an origin.
+
+This function is experimental, and its output is not guaranteed
+to be compatible across different versions of H3.
+
+## experimentalLocalIjToH3
+
+```
+int experimentalLocalIjToH3(H3Index origin, const CoordIJ *ij, H3Index *out);
+```
+
+Produces an H3 index from local IJ coordinates anchored by an origin.
+
+This function is experimental, and its output is not guaranteed
+to be compatible across different versions of H3.
+
+## h3Distance
```

Comment:
This is a little tricky. `h3Distance` is a generally applicable function that many users may want without understanding a concept of local coordinates. I'm a little worried that we're burying it here because its implementation is based on IJ coords, rather than putting it somewhere that would make more sense to the reader.

Would it make more sense, for example, to rename `Neighbors` to `Traversal` and put this there?

Reply:
I agree, organizing this is not easy. While they are not connected from a use perspective, these functions have a similar set of limitations, which I wanted to make the user aware of.

I feel like if `Neighbors` is renamed to `Traversal` that could also encompass the `Local IJ` functions. Maybe all of them should be on the same page?

Reply:
Sure, I'd support that.